nomad-aws/output.tf

```diff
@@ -39,7 +39,8 @@ output "nomad_asg_user_access_key" {
 }
 
 output "nomad_asg_user_secret_key" {
-  value = local.autoscaler_type == "user" ? aws_iam_access_key.nomad_asg_user[0].secret : ""
+  sensitive = true
```

Comment:
This was picked up as an error by `terraform test`. Seems like a valid thing to mark as sensitive...